src/Esfa.Ofsted.Inspection.Client/Services/GetOfstedDetailsFromExcelPackageService.cs

```diff
@@ -58,19 +56,25 @@ public InspectionOutcomesResponse ExtractOfstedInspections(ExcelPackage package)
                 throw exception;
             }
 
-            var lineNumberStart = FindStartingLineNumber(keyWorksheet);
+            var spreadsheetDetails = GetSpreadsheetColumnAndRowDetails(keyWorksheet);
+           
 
-            if (lineNumberStart == 0)
+            if (spreadsheetDetails.WebLinkColumn== 0
+                || spreadsheetDetails.UkPrnColumn == 0
+                || spreadsheetDetails.DatePublishedColumn == 0
+                || spreadsheetDetails.OverallEffectivenessColumn == 0
+                || spreadsheetDetails.DataStartsRow == 0)
             {
                 const string message = "No details could be found when processing";
```

Comment:
can we change it to 
`const string message = "Could not find the start row or the column names;`